frontend/public/components/dashboards-page/with-dashboard-resources.tsx

```diff
@@ -31,8 +30,8 @@ const mapStateToProps = (state: RootState) => ({
   [RESULTS_TYPE.PROMETHEUS]: state.dashboards.get(RESULTS_TYPE.PROMETHEUS),
 });
 
-const WithDashboardResources = (WrappedComponent: React.ComponentType<DashboardItemProps>) =>
-  class WithDashboardResources_ extends React.Component<WithDashboardResourcesProps, WithDashboardResourcesState> {
+export const withDashboardResources = <P extends DashboardItemProps>(WrappedComponent: React.ComponentType<P>, additionalProps: any = {}) => (
+  connect(mapStateToProps, mapDispatchToProps)(class WithDashboardResources_ extends React.Component<WithDashboardResourcesProps, WithDashboardResourcesState> {
```

Comment:
Nit: for better readability, please make the `class WithDashboardResources_` declaration start on a new line.

Reply:
moved to new line. I also removed  `_` from `class WithDashboardResources_` as its not needed